### ISSUE TITLE
List pubs randomly

### ISSUE THREAD

**Issue by RohrerHope** [Whichost/frontend#83]
![screen shot 2019-02-16 at 14 50 31](https://user-images.githubusercontent.com/39112974/52901388-bc2a8900-31fa-11e9-8075-5b6cb6d1d7f2.jpg)

Pubs in the list are clearly been shown from newest (newest listing created) to oldest (oldest listing created).

This does not give the users an experience of seeing regularly new pubs on the list. Because the newest are always on the top.

Can we list pubs in a random way when we click on Browse pubs?

Maybe we can apply the same logic to when a user applies a filter? So the pubs that match the filters won't be displayed in a specific order, but a random one.

**Comment by lcoenen** [comment]
I guess we could, but isn't that a little dangerous? I mean the "newer" order isn't optimal obviously, but shouldn't we weight them differently according to some criterias that we will define?

**Comment by RohrerHope** [comment]
Definitely. We should have a set of rules that will define how pubs are shown.

Now, for now isn't it easier to set it up as random? And then we figure out a better model?

This way at least users will not always see the same pubs at the top of the list.

**Comment by lcoenen** [comment]
I will see what's possible to do, but I am not sure I have control over what sharetribe is sending. Stay tuned.

**Comment by RohrerHope** [comment]
Thanks.

**Comment by lcoenen** [comment]
![image](https://user-images.githubusercontent.com/1237971/54436498-8cec3680-472a-11e9-96c2-20d0edc9cd7c.png)
